<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PWSlotUD</class>
 <widget class="QWidget" name="PWSlotUD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="MPLCanvas" name="w_viewer" native="true"/>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>270</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>270</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>268</width>
        <height>416</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="b_dxf">
         <property name="text">
          <string>Define Slot from DXF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="WPathSelectorV" name="w_path_json" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="g_wedge">
         <property name="title">
          <string>Wedge</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="WMatSelectV" name="w_wedge_mat" native="true">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="WWSlotOut" name="w_out" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WWSlotOut</class>
   <extends>QWidget</extends>
   <header>......GUI.Dialog.DMachineSetup.SWSlot.WWSlotOut.WWSlotOut.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MPLCanvas</class>
   <extends>QWidget</extends>
   <header>......GUI.Tools.MPLCanvas</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WMatSelectV</class>
   <extends>QWidget</extends>
   <header>......GUI.Dialog.DMatLib.WMatSelect.WMatSelectV.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WPathSelectorV</class>
   <extends>QWidget</extends>
   <header>......GUI.Tools.WPathSelector.WPathSelectorV</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../Resources/pyleecan.qrc"/>
  <include location="../../../../Resources/pyleecan.qrc"/>
 </resources>
 <connections/>
</ui>
